### PR TITLE
Move language code into a subpackage

### PR DESCRIPTION
**Pull request by dhermes**
This PR was created via: https://gist.github.com/dhermes/e239691aa584bd56a5352e34aad27cf3

```
export PROJECT_DIR="${HOME}/google-cloud-python"
export READMES_DIR="${HOME}/i-wrote-some-readmes-for-2357"
cd ${PROJECT_DIR}
git worktree add -b make-language-subpackage ../hotfix official/master
python make_commits.py \
  --git-root "${PROJECT_DIR}/../hotfix" \
  --package language \
  --package-name "Google Cloud Natural Language" \
  --readme "${READMES_DIR}/language/README.rst"
```

### CODE REVIEW

File: language/README.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+.. _API Documentation: http://googlecloudplatform.github.io/google-cloud-python/
+
+Quick Start
+-----------
+
+::
+
+    $ pip install --upgrade google-cloud-language
+    $ # OR
+    $ pip install --upgrade google-cloud-natural-language
```
Comment (jonparrott) [inline language/README.rst]: I don't see how this alt package is published?

Comment (dhermes) [inline language/README.rst]: It isn't covered here. Adding to #2404.

**Comment by dhermes** [comment]
For some reason this didn't kick off a new Travis build when I rebased on top of #2428. Maybe the "approval" stops it?

**Comment by dhermes** [comment]
Here is the orignal (green) PR build: https://travis-ci.org/GoogleCloudPlatform/google-cloud-python/builds/162895936

**Pull request merged**